index.rst

```diff
@@ -0,0 +1,28 @@
+Dask Tutorial
+=============
+
+You can run this tutorial in a live session here: |Binder|
+
+This tutorial was last given at SciPy 2018 in Austin Texas.
+`A video is available online <https://www.youtube.com/watch?v=mqdglv9GnM>`_.
```

Comment:
I wonder there is a way to get this YouTube video in a JupyterLab pane when the user starts up their session.  Any suggestions @ian-r-rose?

Reply:
It should be possible -- the help panels are just iframes on the page, and those are serialized in workspace files. If you open a help panel, save the workspace, and manually replace the url in the workspace JSON with the youtube embed link, it should do the job.